apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/RdaService.java

```diff
@@ -16,22 +20,32 @@
  */
 public class RdaService extends RDAServiceGrpc.RDAServiceImplBase {
   private static final Logger LOGGER = LoggerFactory.getLogger(RdaService.class);
-  private final MessageSource.Factory<FissClaimChange> fissSourceFactory;
-  private final MessageSource.Factory<McsClaimChange> mcsSourceFactory;
-
-  public RdaService(
-      MessageSource.Factory<FissClaimChange> fissSourceFactory,
-      MessageSource.Factory<McsClaimChange> mcsSourceFactory) {
-    this.fissSourceFactory = fissSourceFactory;
-    this.mcsSourceFactory = mcsSourceFactory;
+  public static final String RDA_PROTO_VERSION = "0.4";
```

Comment:
is this only needed for testing?  when running the server all versions would either be `S3:*` or `Random:*` right?